<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaMenuPrincipal</class>
 <widget class="QMainWindow" name="ventanaMenuPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1900</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1900</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Men√∫ principal del sistema</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelFondoMenuConfiguracion">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1900</width>
      <height>850</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>menuConfiguracionFigma.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="labelBorrarCurriculums">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>591</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelBorrarDatos">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>230</y>
      <width>591</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelVolverInicio">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>0</y>
      <width>421</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelActualizarPorcentaje">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>480</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>400</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>40</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>60</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>70</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>80</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>90</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
